.github/workflows/dev-build-to-pypi.yml

```diff
@@ -0,0 +1,69 @@
+name: Dev build and publish Python distributions to PyPI
```

Comment:
Could you please use common workflow for publish python tarball

Reply:
Please look at the example
https://github.com/th2-net/th2-data-services/blob/dev_2.0.0/.github/workflows/publish-snapshot.yml